src/test/java/com/scottlogic/TopicFinderTest.java

```diff
@@ -0,0 +1,54 @@
+package com.scottlogic;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public  class TopicFinderTest {
+
+
+    private static List<String> stopWords;
+
+    private UserPost createUserPost(String content) {
+        return new UserPost("Joe Bloggs",
+                OffsetDateTime.of(2020, 1, 3, 7, 12, 3, 0, ZoneOffset.UTC),
+                content, 2);
+    }
+
+    @BeforeClass
+    public static void loadStopWords() throws IOException {
+        File file = new File("C:/Dev/post-sorting-grad-project/src/main/resources/English_StopWords.txt");
+       stopWords = Files.readAllLines(Paths.get(String.valueOf(file)));
+    }
+
+    TopicFinder topicFind = new TopicFinder(stopWords);
+
+    @Test
+    public void topicFinder_generalPost_stopWordsPunctuationRemoved()  {
```

Comment:
This might be a good case for looking into [parameterised tests](https://www.baeldung.com/parameterized-tests-junit-5). We could set up a list of strings, each with a single thing to remove (a different stop word or punctuation character), then test each one in turn without having to write hundreds of test methods.